src/dotnet/commands/dotnet-new/CSharp_Lib/Library.cs

```diff
@@ -4,8 +4,5 @@ namespace ClassLibrary
 {
     public class Class1
     {
-        public void Method1()
-        {
-        }
```

Comment:
we're unifying templates across distribution mechanisms and the current source templates do not include a method. @davkean may have more info on this.